.cspell.json

```diff
@@ -31,11 +34,13 @@
     "kubeapps",
     "kubes",
     "kubeval",
+    "madr",
     "millis",
     "mkilled",
     "nindent",
     "nothrow",
     "nslookup",
+    "nygard",
```

Comment:
what are these keywords? I don't recognise them. If it they are not part of our core tech stack, we should not be including them :)

Reply:
Spelling words. Used in code.